tests/e2e/mapper/bluetooth/bluetooth_suite_test.go

```diff
@@ -0,0 +1,270 @@
+		Expect(utils.StartEdgeController()).Should(BeNil())
+
+		//Register the Edge Node to Master
+		err = utils.RegisterNodeToMaster(nodeName, ctx.Cfg.K8SMasterForKubeEdge+NodeHandler, nodeSelector)
+		Expect(err).Should(BeNil())
+
+		//Run ./edge_core after node registration
+		Expect(utils.StartEdgeCore()).Should(BeNil())
+
+		//Check node successfully registered or not
+		Eventually(func() string {
+			status := utils.CheckNodeReadyStatus(ctx.Cfg.K8SMasterForKubeEdge+NodeHandler, nodeName)
+			utils.Info("Node Name: %v, Node Status: %v", nodeName, status)
+			return status
+		}, "60s", "4s").Should(Equal("Running"), "Node register to the k8s master is unsuccessfull !!")
+		fmt.Println("Node in Ready State")
+
+		// Adding label to node
+		utils.ApplyLabelToNode(ctx.Cfg.K8SMasterForKubeEdge+NodeHandler+"/"+nodeName, "bluetooth", "true")
+
+		// Changing the config yaml of bluetooth mapper
+		t = time.Now()
+		pwd := getpwd()
+		sourcepath := path.Join(pwd, "./configuration/config.yaml")
+		destinationpath := path.Join(pwd, "../../../../device/bluetooth_mapper/configuration/config.yaml")
+		cmd := exec.Command("cp", sourcepath, destinationpath)
+		err = utils.PrintCombinedOutput(cmd)
+		Expect(err).Should(BeNil())
+
+		//Building bluetooth mapper
+		curpath = getpwd()
+		newpath := path.Join(curpath, "../../../../device/bluetooth_mapper/")
+		os.Chdir(newpath)
+		cmd = exec.Command("make", "bluetooth_mapper_image")
+		err = utils.PrintCombinedOutput(cmd)
+		Expect(err).Should(BeNil())
+
+		//dockertag
+		tagname := cfg.DockerHubUserName + "/bluetooth_mapper:v1.0"
+		cmd = exec.Command("docker", "tag", "bluetooth_mapper:v1.0", tagname)
+		err = utils.PrintCombinedOutput(cmd)
+		Expect(err).Should(BeNil())
+
+		//docker login
+		cmd = exec.Command("docker", "login", "-u", cfg.DockerHubUserName, "-p", cfg.DockerHubPassword)
+		err = utils.PrintCombinedOutput(cmd)
+		Expect(err).Should(BeNil())
+
+		//docker push
+		cmd = exec.Command("docker", "push", tagname)
+		err = utils.PrintCombinedOutput(cmd)
+		Expect(err).Should(BeNil())
+
+		//apply CRD for mock devicemodel
+		curpath = getpwd()
+		//file := path.Join(curpath, "../../../../build/crd-samples/devices/devicemockmodel.yaml")
+		file = path.Join(curpath, "./crds/devicemodel.yaml")
+		body, err = ioutil.ReadFile(file)
+		if err == nil {
+			client := &http.Client{}
+			BodyBuf := bytes.NewReader(body)
+			req, err := http.NewRequest(http.MethodPost, ctx.Cfg.K8SMasterForKubeEdge+MockHandler, BodyBuf)
+			Expect(err).Should(BeNil())
+			req.Header.Set("Content-Type", "application/yaml")
+			resp, err := client.Do(req)
+			Expect(err).To(BeNil())
+			Expect(resp.StatusCode).Should(Equal(http.StatusCreated))
+			utils.InfoV6("%s %s %v in %v", req.Method, req.URL, resp.Status, time.Now().Sub(t))
+		}
+
+		//apply CRD for mock deviceinstance
+		curpath = getpwd()
+		file = path.Join(curpath, "./crds/deviceinstance.yaml")
+		body, err = ioutil.ReadFile(file)
+		if err == nil {
```

Comment:
Can we replace this if with "Expect(err).Should(BeNil())"

Reply:
Updated